### DISCUSSION TITLE
Scrolling Text

### DISCUSSION THREAD

**Discussion by marshyrob** [BimmerGestalt/AAIdrive#533] [Ideas]
Hi Again,

Is there a way to make the song text name scroll.  What i mean by this is, that sometimes my song is called something like,

**abcdefghijklmnopkrstuvwxyz - Remix**

But all i will see on the screen due to width limit when split screen is on on:

**abcdefghijklmnopkrs**

So can the text be made to scroll left to show the whole name at least once maybe when the song first displays?

Cheers

Rob

**Comment by hufman** [comment]
That's a good idea! The car itself doesn't seem to have any built-in functionality like this, but it should be easy enough to implement.

**Reply by hufman** [reply]
I've created a feature request at #534
